{"uid":"87e58bad754f8d44","name":"test_reelection","fullName":"tests.integration.ha_tests.test_replication#test_reelection","historyId":"c4fc3ed629099ecebc757c8005dc7207","time":{"start":1729042485225,"stop":1729042485225,"duration":0},"description":"Kill primary unit, check reelection.","descriptionHtml":"<p>Kill primary unit, check reelection.</p>\n","status":"broken","statusMessage":"tenacity.RetryError: RetryError[<Future at 0x7fcb2b79e110 state=finished raised AttributeError>]","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 441, in get_patroni_setting\n    unit_ip = get_unit_address(ops_test, primary_name)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 336, in wrapped_f\n    return copy(f, *args, **kw)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 475, in __call__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 376, in iter\n    result = action(retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 418, in exc_check\n    raise retry_exc.reraise()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 185, in reraise\n    raise self.last_attempt.result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 478, in __call__\n    result = fn(*args, **kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/helpers.py\", line 697, in get_unit_address\n    return model.units.get(unit_name).public_address\nAttributeError: 'NoneType' object has no attribute 'public_address'\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 302, in _asyncgen_fixture_wrapper\n    result = event_loop.run_until_complete(setup())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 284, in setup\n    res = await gen_obj.__anext__()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/ha_tests/conftest.py\", line 64, in primary_start_timeout\n    initial_primary_start_timeout = await get_patroni_setting(ops_test, \"primary_start_timeout\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 437, in get_patroni_setting\n    for attempt in Retrying(stop=stop_after_delay(30 * 2), wait=wait_fixed(3)):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 443, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 376, in iter\n    result = action(retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 419, in exc_check\n    raise retry_exc from fut.exception()\ntenacity.RetryError: RetryError[<Future at 0x7fcb2b79e110 state=finished raised AttributeError>]","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1729042236751,"stop":1729042236752,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1729042236752,"stop":1729042236752,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1729042236752,"stop":1729042236903,"duration":151},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout","time":{"start":1729042485226,"stop":1729042650574,"duration":165348},"status":"broken","statusMessage":"tenacity.RetryError: RetryError[<Future at 0x7fcb2b79e110 state=finished raised AttributeError>]\n","statusTrace":"  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 302, in _asyncgen_fixture_wrapper\n    result = event_loop.run_until_complete(setup())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 284, in setup\n    res = await gen_obj.__anext__()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/ha_tests/conftest.py\", line 64, in primary_start_timeout\n    initial_primary_start_timeout = await get_patroni_setting(ops_test, \"primary_start_timeout\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 437, in get_patroni_setting\n    for attempt in Retrying(stop=stop_after_delay(30 * 2), wait=wait_fixed(3)):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 443, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 376, in iter\n    result = action(retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 419, in exc_check\n    raise retry_exc from fut.exception()\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1729042650580,"stop":1729042650580,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1729042650584,"stop":1729042650894,"duration":310},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1729042650894,"stop":1729042650894,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1729042650895,"stop":1729042650895,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1729042650895,"stop":1729042650895,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_replication"},{"name":"host","value":"fv-az841-587"},{"name":"thread","value":"15917-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_replication"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"7ed5b736b01671a5","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":53,"skipped":2,"passed":71,"unknown":4,"total":130},"items":[{"uid":"8a1e8eb346f4b424","reportUrl":"../205//#testresult/8a1e8eb346f4b424","status":"passed","time":{"start":1728956444417,"stop":1728956582318,"duration":137901}},{"uid":"2e0e0017b7ab8821","reportUrl":"../204//#testresult/2e0e0017b7ab8821","status":"passed","time":{"start":1728870115480,"stop":1728870274166,"duration":158686}},{"uid":"5441989a8282ed99","reportUrl":"../203//#testresult/5441989a8282ed99","status":"passed","time":{"start":1728783652962,"stop":1728783810467,"duration":157505}},{"uid":"24fa1c93eff1f2d2","reportUrl":"../202//#testresult/24fa1c93eff1f2d2","status":"passed","time":{"start":1728697260298,"stop":1728697418507,"duration":158209}},{"uid":"2d54f50ced48f10b","reportUrl":"../201//#testresult/2d54f50ced48f10b","status":"passed","time":{"start":1728610874268,"stop":1728611016979,"duration":142711}},{"uid":"b08008f800ea2e8e","reportUrl":"../200//#testresult/b08008f800ea2e8e","status":"passed","time":{"start":1728524491733,"stop":1728524654243,"duration":162510}},{"uid":"bc53e7e61dc70e9f","reportUrl":"../199//#testresult/bc53e7e61dc70e9f","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728437781345,"stop":1728437781345,"duration":0}},{"uid":"291e1cce4cc2505f","reportUrl":"../198//#testresult/291e1cce4cc2505f","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728351418149,"stop":1728351418149,"duration":0}},{"uid":"7f92ea313046bcb2","reportUrl":"../197//#testresult/7f92ea313046bcb2","status":"passed","time":{"start":1728265272736,"stop":1728265431899,"duration":159163}},{"uid":"a3f21bba59a59099","reportUrl":"../196//#testresult/a3f21bba59a59099","status":"passed","time":{"start":1728178822233,"stop":1728178969014,"duration":146781}},{"uid":"8936a1a2f570ddef","reportUrl":"../195//#testresult/8936a1a2f570ddef","status":"passed","time":{"start":1728092486307,"stop":1728092647829,"duration":161522}},{"uid":"8a0152701f0f23e5","reportUrl":"../194//#testresult/8a0152701f0f23e5","status":"passed","time":{"start":1728006009225,"stop":1728006166211,"duration":156986}},{"uid":"25c8e7c0ffa6c4cb","reportUrl":"../193//#testresult/25c8e7c0ffa6c4cb","status":"passed","time":{"start":1727919855285,"stop":1727920007329,"duration":152044}},{"uid":"5fccc8770303a187","reportUrl":"../192//#testresult/5fccc8770303a187","status":"passed","time":{"start":1727833301310,"stop":1727833466626,"duration":165316}},{"uid":"a5a3aba175aeda16","reportUrl":"../191//#testresult/a5a3aba175aeda16","status":"passed","time":{"start":1727746867307,"stop":1727747011017,"duration":143710}},{"uid":"74dad76bee9f49d9","reportUrl":"../190//#testresult/74dad76bee9f49d9","status":"passed","time":{"start":1727660441747,"stop":1727660592336,"duration":150589}},{"uid":"641425b03fae9c2","reportUrl":"../189//#testresult/641425b03fae9c2","status":"passed","time":{"start":1727574018166,"stop":1727574162782,"duration":144616}},{"uid":"63d6cd208a0fec2b","reportUrl":"../188//#testresult/63d6cd208a0fec2b","status":"passed","time":{"start":1727487689051,"stop":1727487853582,"duration":164531}},{"uid":"6db131fbf5324056","reportUrl":"../187//#testresult/6db131fbf5324056","status":"passed","time":{"start":1727401211213,"stop":1727401371214,"duration":160001}},{"uid":"37108c424c8e9542","reportUrl":"../186//#testresult/37108c424c8e9542","status":"passed","time":{"start":1727314931708,"stop":1727315073657,"duration":141949}}]},"tags":["asyncio","group(1)"]},"source":"87e58bad754f8d44.json","parameterValues":[]}